zaza/openstack/charm_tests/octavia/tests.py

```diff
@@ -519,3 +521,42 @@ def test_create_loadbalancer(self):
 
         # If we get here, it means the tests passed
         self.run_resource_cleanup = True
+
+
+@unittest.skipIf(openstack_utils.get_os_release() <
+                 openstack_utils.get_os_release('bionic_train'),
```

Comment:
The charm option is introduced from Ussuri https://review.opendev.org/c/openstack/charm-octavia/+/810567.
Is this check supposed to be bionic_ussuri?

Reply:
Yes, the feature should be introduced at Ussuri. Not sure why I had this as bionic_train.